### PR TITLE
improved the automatic bed generation

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-germline/blob/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -25,9 +25,8 @@ process {
     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     */
 
-    withName: INDEX_TO_BED {
-        ext.args2   = '\'BEGIN {FS="\t"}; {print \$1 FS "0" FS \$2}\''
-        ext.suffix  = 'bed'
+    withName: MOSDEPTH {
+        ext.args = "--no-per-base --fast-mode --quantize 0:1:4:"
```
Comment (matthdsm) [inline conf/modules.config:29]:
```suggestion
        ext.args = "--no-per-base --fast-mode --quantize 0:1:4: --flag 1804 --mapq 1" 
```
Filter reads by flag and mapq

Comment (matthdsm) [reply conf/modules.config:29]: I've also added the required env variables at the bottom of `modules.conf`

Comment (matthdsm) [reply conf/modules.config:29]: To correctly determine variant calling regions, `quantized` bed should be filtered on regions with the `CALLABLE` tag and then intersected with an optional given ROI

Comment (nvnieuwk) [reply conf/modules.config]: Allright thanks!

**Pull request merged**